src/signals/incident-management/containers/IncidentDetail/components/Attachments/Attachments.tsx

```diff
@@ -236,6 +236,7 @@ const Attachments: FC<AttachmentsProps> = ({
                     <StyledButton
                       icon={<DeleteIcon />}
                       iconSize={18}
+                      data-testid={attachment.location + attachment.created_at}
```

Comment:
this will be a really strange data test id name, consider something more humanlike?

Reply:
I've changed the name to DeleteIcon + attachment.created_at. It does need the attachment.created_at in order to be unique.

Reply:
yeah or just attachment + index